### PR TITLE
Add API the DOcs to the front page, and clean up copy

### PR DESCRIPTION
**Pull request by ericholscher**

### CODE REVIEW

File: docs/include/conf/current.rst [hunk 1]
```diff
@@ -15,12 +15,16 @@
 
 - :doc:`Write the Docs Day Australia </conf/au/2017/index>`, November 24, **Melbourne, Australia**
 
+Want to run your own one-day Write the Docs event? Check our :doc:`guide for starting an event </organizer-guide/confs/start/>` and `email us <mailto:conf@writethedocs.org>`_ if you have any questions!
```
Comment (thatdocslady) [inline docs/include/conf/current.rst]: I thought we're retiring the conf@ address in favor of support@ and city-specific? (on a related note: we should go over the website and update all the addresses.

File: docs/include/conf/current.rst [hunk 2]
```diff
@@ -15,12 +15,16 @@
 
 - :doc:`Write the Docs Day Australia </conf/au/2017/index>`, November 24, **Melbourne, Australia**
 
+Want to run your own one-day Write the Docs event? Check our :doc:`guide for starting an event </organizer-guide/confs/start/>` and `email us <mailto:conf@writethedocs.org>`_ if you have any questions! 
+
 2017 Community Conferences
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Think your conference should go here? Make sure you read our :doc:`/organizer-guide/confs/community-events` page and meet all the requirements, and then email us conf@writethedocs.org. 
+- `API The Docs Amsterdam <http://apithedocs.org/amsterdam/>`_, December 4, **Amsterdam, Netherlands**
```
Comment (thatdocslady) [inline docs/include/conf/current.rst]: The > the (style guide?)

File: docs/include/conf/current.rst [hunk 3]
```diff
@@ -15,12 +15,16 @@
...
+
+Think your conference should go here? Make sure you read our :doc:`/organizer-guide/confs/community-events` page and meet all the requirements, and then `email us <mailto:conf@writethedocs.org>`_.
```
Comment (thatdocslady) [inline docs/include/conf/current.rst]: maybe rephrase to "make sure your event meets the requirements listed in <link to page>"

**Review by thatdocslady — COMMENTED** [review]
Looks good, some nits :)

File: docs/include/conf/current.rst (hunk 1 above)
Comment (ericholscher) [reply docs/include/conf/current.rst]: Aye, I just copied it from below -- it makes some sense for this case, but yea should standardize.

File: docs/include/conf/current.rst (hunk 2 above)
Comment (ericholscher) [reply docs/include/conf/current.rst]: They are using it cap'd -- so we'd need to talk with them about it probably.
